README.md

```diff
@@ -21,127 +47,131 @@ c2patool is a command line tool for working with C2PA [manifests](https://c2pa.o
+Use the `--remote` / `-r` option to places an HTTP reference to the manifest in the output file. The manifest is returned as an external sidecar file in the same location as the output file with the same filename but with a `.c2pa` extension. Place the manifest at the location specified by the `-r` option. When using remote manifests the remote URL should be publicly accessible to be most useful to users. When verifying an asset, remote manifests are automatically fetched. 
 
 ```shell
 c2patool sample/image.jpg -r http://my_server/myasset.c2pa -m sample/test.json -o signed_image.jpg
 ```
 
-In the example above c2patool will try to fetch the manifest for new_manifest.jpg from http://my_server/myasset.c2pa during validation.
+In the example above, the tool will embed the URL http://my_server/myasset.c2pa in `signed_image.jpg` then fetch the manifest from that URL and save it to `signed_image.c2pa`.
 
-Note: It is possible to combine the `-s` and `-r` options. When used together a manifest will be embedded in the output files and the remote reference will also be added. 
+If you use both the `-s` and `-r` options, the tool embeds a manifest in the output file and also adds the remote reference.
 
-#### Example of a manifest definition file
-
-Here's an example of a manifest definition that inserts a CreativeWork author assertion. If you copy this into a JSON file, you can use it as a test manifest definition.
-
-```json
-{
-    "ta_url": "http://timestamp.digicert.com",
-    
-    "claim_generator": "TestApp",
-    "assertions": [
-        {
-            "label": "stds.schema-org.CreativeWork",
-            "data": {
-                "@context": "https://schema.org",
-                "@type": "CreativeWork",
-                "author": [
-                    {
-                        "@type": "Person",
-                        "name": "Joe Bloggs"
-                    }
-                ]
-            }
-        }
-    ]
-}
-```
-### Manifest definition on command line
+### Providing a manifest definition on the command line
 
-The [manifest definition](#manifest-definition-format) can also be passed on the command line as a string using the `-c` or `--config` option.
+To provide the [manifest definition](#manifest-definition-file) in a command line argument instead of a file, use the `--config` / `-c` option.
 
-In this example we are adding a custom assertion called "org.contentauth.test".
+For example, the following command adds a custom assertion called "org.contentauth.test".
 
 ```shell
-c2patool sample/image.json -c '{"assertions": [{"label": "org.contentauth.test", "data": {"my_key": "whatever I want"}}]}'
+c2patool -c '{"assertions": [{"label": "org.contentauth.test", "data": {"my_key": "whatever I want"}}]}'
```

Comment:
This has to have an asset path

c2patool sample/image.json -c '{"assertions": [{"label": "org.contentauth.test", "data": {"my_key": "whatever I want"}}]}'